cpp/include/raft/sparse/distance/distance.cuh

```diff
@@ -130,8 +132,101 @@ void pairwiseDistance(value_t* out,
   }
 }
 
-};  // namespace distance
-};  // namespace sparse
-};  // namespace raft
+/**
+ * @defgroup sparse_distance Sparse Pairwise Distance
+ * @{
+ */
+
+/**
+ * @brief Compute pairwise distances between x and y, using the provided
+ * input configuration and distance function.
+ *
+ * @code{.cpp}
+ * #include <raft/core/device_resources.hpp>
+ * #include <raft/core/device_coo_matrix.hpp>
+ * #include <raft/core/device_mdspan.hpp>
+ *
+ * int x_n_rows = 100000;
+ * int y_n_rows = 50000;
+ * int n_cols = 10000;
+ *
+ * raft::device_resources handle;
+ * auto x = raft::make_device_csr_matrix<float>(handle, x_n_rows, n_cols);
+ * auto y = raft::make_device_csr_matrix<float>(handle, y_n_rows, n_cols);
+ * ...
+ * // compute expected sparsity
+ * ...
+ * int x_nnz = 5000;
+ * int y_nnz = 10000;
+ * x.initialize_sparsity(nnz);
```

Comment:
Maybe for purposes of calling the pairwise distances API, we can skip this step and just have the `// populate data` comment?